src/components/Person/Person.jsx

```diff
@@ -1 +1,32 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  let partnerText;
+  let partnerAge;
+
+  if (person.isMarried) {
+    if (person.sex === 'm') {
+      partnerText = (
+        <p className="Person__partner">{person.partnerName} is my wife</p>
+      );
+    } else {
+      partnerText = (
+        <p className="Person__partner">{person.partnerName} is my husband</p>
+      );
+    }
+  } else {
+    partnerText = <p className="Person__partner">I am not married</p>;
+  }
```

Comment:
Separate these conditions into separate variables and use the ternary operator

Reply:
I didn't use ternary operator because linter made my code red and said that I can't use ternary